tests/docker-compose.yml

```
@@ -3,9 +3,8 @@ services:
     build:
       context: ..
       dockerfile: ./Dockerfile.test
-    command: >
-      go test -v -coverprofile=coverage/coverage.out -covermode=atomic ./... && \
-        go tool cover -html=coverage/coverage.out -o coverage/coverage.html
+    command: >  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)